src/inputs/button/button.jsx

```diff
@@ -2,25 +2,172 @@ import {
+    as: PropTypes.oneOf(Object.values(AsType)),
+    /**
+     * Primary content.
+     */
+    children: PropTypes.node.isRequired,
+    /**
+     * Additional classes.
+     */
+    className: PropTypes.string,
+    /**
+     * Override or extend the styles applied to ButtonDropdown.
+     */
+    classes: PropTypes.shape({
+        colorActive: PropTypes.string,
+        colorDefault: PropTypes.string,
+        colorError: PropTypes.string,
+        colorLink: PropTypes.string,
+        colorPrimary: PropTypes.string,
+        colorSecondary: PropTypes.string,
+        colorSuccess: PropTypes.string,
+        colorWarning: PropTypes.string,
+        compact: PropTypes.string,
+        contained: PropTypes.string,
+        disabled: PropTypes.string,
+        fixedWidth: PropTypes.string,
+        fullWidth: PropTypes.string,
+        icon: PropTypes.string,
+        innerContainer: PropTypes.string,
+        inverse: PropTypes.string,
+        outlined: PropTypes.string,
+        pill: PropTypes.string,
+        relax: PropTypes.string,
+        root: PropTypes.string,
+        text: PropTypes.string,
+        transparent: PropTypes.string,
+    }),
+    /**
+     * Color of the button.
+     */
+    color: PropTypes.oneOf(Object.values(ColorType)),
+    /**
+     * A button can reduce its padding.
+     */
+    compact: PropTypes.bool,
+    /**
+     * A button can be disabled.
+     */
+    disabled: PropTypes.bool,
+    /**
+     * The Button will be resized to its parent container's width.
+     */
+    fullWidth: PropTypes.bool,
+    /**
+     * The URL that the hyperlink points to.
+     */
+    href: PropTypes.string,
+    /**
+     * If `true`, the button will be a square, housing the icon child.
+     */
+    icon: PropTypes.bool,
+    /**
+     * Assign the button an id attribute value.
+     */
+    id: PropTypes.string,
+    /**
+     * Allows for style overrides of the Button's inner container.
+     */
+    innerStyle: PropTypes.shape({}),
+    /**
+     * A button can be formatted to appear on dark backgrounds better.
+     */
+    inverse: PropTypes.bool,
+    /**
+     * The onClick event handler.
+     */
+    onClick: PropTypes.func,
+    /**
+     * A button can be outlined.
+     */
+    outline: PropTypes.bool,
+    /**
+     * Set a button with a pill like form.
+     */
+    pill: PropTypes.bool,
+    /**
+     * A button can relax its padding.
+     */
+    relax: PropTypes.bool,
+    /**
+     * A button can relax its padding.
+     */
+    style: PropTypes.shape({}),
+    /**
+     * Where to display the linked URL.
+     */
+    target: PropTypes.oneOf(['_blank']),
+    /**
+     * If `true`, only the button's text is shown.
+     */
+    text: PropTypes.bool,
+    /**
+     * The title attribute.
+     */
+    title: PropTypes.string,
+    /**
+     * Set transparent styles.
+     */
+    transparent: PropTypes.bool,
+    /**
+     * Set transparent styles.
+     */
+    type: PropTypes.oneOf(Object.values(Type)),
```

Comment:
Let's fix up this documentation comment (which appears to be copied and pasted) and maybe link to that MDN page about browser's button attribute.

Reply:
Resolved in a9ef7f5.